controller/router.js

```diff
@@ -95,6 +97,16 @@ module.exports = [
                         .number()
                         .integer()
                         .positive()
+                },
+                query: {
+                    fromDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD')
+                       .default(moment().startOf('day').toDate()),
+                    toDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD')
+                       .default((moment().add(addSevenDays, 'days')).toDate())
```

Comment:
I'll do this. But as far as I remember Dr. Gary told it put it as (x) and (x+7) rather